src/frontend/src/flows/authenticate/index.ts

```diff
@@ -43,6 +43,15 @@ export const authenticationFlow = async (
 ): Promise<void> => {
   const container = document.getElementById("pageContent") as HTMLElement;
   render(html`<h1>starting authentication</h1>`, container);
+  const showMessage = (msg: string) =>
+    render(
+      html`<h1
+        style="position: absolute; max-width: 100%; top: 50%; transform: translate(0, -50%);"
```

Comment:
Wouldn't it be nicer to extract the styling and add a class for that?

Reply:
these are transient screens, not really part of the design system, so I thought it'd be nicer to keep it self contained